packages/callhierarchy/src/browser/callhierarchy-tree/callhierarchy-tree-widget.tsx

```diff
@@ -17,17 +17,18 @@
 import { injectable, inject } from "inversify";
 import { Message } from "@phosphor/messaging";
 import {
-    ContextMenuRenderer, TreeWidget, NodeProps, TreeProps, TreeNode,
+    ContextMenuRenderer, NodeProps, TreeProps, TreeNode,
     SelectableTreeNode, TreeModel, DockPanel
 } from "@theia/core/lib/browser";
-import { ElementAttrs, h } from "@phosphor/virtualdom";
 import { LabelProvider } from "@theia/core/lib/browser/label-provider";
 import { DefinitionNode, CallerNode } from "./callhierarchy-tree";
 import { CallHierarchyTreeModel } from "./callhierarchy-tree-model";
 import { CALLHIERARCHY_ID, Definition, Caller } from "../callhierarchy";
 import URI from "@theia/core/lib/common/uri";
 import { Location, Range, SymbolKind } from 'vscode-languageserver-types';
 import { EditorManager } from "@theia/editor/lib/browser";
+import { TreeWidget } from "@theia/core/lib/browser/tree/tree-widget";
```

Comment:
Same as above. Please do not change the imports if not necessary.

Reply:
Same as above. Was tree-react-widget temporarily.